lib/patches/active_record/rails7_1/serialization.rb

```diff
@@ -0,0 +1,28 @@
+module Globalize
+  module AttributeMethods
+    module Serialization
+      def serialize(attr_name, class_name_or_coder = nil, **options)
```

Comment:
I think we do not really care for the coder like here https://github.com/globalize/globalize/blob/main/lib/patches/active_record/rails6_1/serialization.rb. we only want to "reuse" the given options in `Globalize::ActiveRecord::ActMacro#enable_serializable_attribute`